layers/+lang/haskell/packages.el

```diff
@@ -29,12 +29,15 @@
   (use-package cmm-mode
     :defer t))
 
+(setq haskell-modes '(haskell-mode literate-haskell-mode))
+
 (when (configuration-layer/layer-usedp 'spacemacs-helm)
   (defun haskell/init-helm-hoogle ()
     (use-package helm-hoogle
       :defer t
       :init
-      (spacemacs/set-leader-keys-for-major-mode 'haskell-mode "hf" 'helm-hoogle))))
+      (dolist (m haskell-modes)
+        (spacemacs/set-leader-keys-for-major-mode m "hf" 'helm-hoogle)))))
 
 (defun haskell/post-init-flycheck ()
   (spacemacs/add-flycheck-hook 'haskell-mode))
```

Comment:
hmm it should, I'm going to be using this a lot tomorrow so I'll be able to test it a bunch